spec/classes/dhcp_spec.rb

```diff
@@ -4,6 +4,7 @@
     {
       'dnsdomain'           => ['sampledomain.com', '1.1.1.in-addr.arpa'],
       'nameservers'         => ['1.1.1.1'],
+      'nameservers_ipv6'    => ['1:5ee:bad:c0de'],
```

Comment:
Why does it need to be a valid ipv6 address? it is only used as a string in the spec test